src/poliastro/bodies.py

```diff
@@ -9,15 +9,22 @@
 and a way to define new bodies (:py:class:`~Body` class).
 
 """
-from astropy.constants import R_earth
+
 from astropy import units as u
+import warnings
+from poliastro.twobody import Orbit
+from poliastro.twobody.angles import M_to_nu
+import os.path
 
+au = 149597870.700 * u.km
+body_dict = {}
 
 class Body(object):
     """Class to represent a body of the Solar System.
 
     """
-    def __init__(self, k, name=None, symbol=None, R=0 * u.km):
+    def __init__(self, k, name=None, symbol=None, R=0 * u.km,
+                 parent=None, orbit = None):
```

Comment:
I still dislike the idea of adding an orbit to the body itself. The mean elements available at a certain epoch are somewhat useful, but having approximate ephemerides through Astropy and ERFA it looks like duplicated data to me. I would still like the `Body` class to be as simple as possible, removing the orbit and the associated imports.